app/models/user.rb

```diff
@@ -1,5 +1,14 @@
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: "Relationship",
```

Comment:
```suggestion
  has_many :microposts, dependent: :destroy
  has_many :active_relationships, class_name: Relationship.name,
```
sao ở đây cần định nghĩa class name và foreign_key em nhỉ

Reply:
Để khi trường hợp không thể lấy được tên của model thì sử dụng class_name để cung cấp tên cho model, foreign_key để cung cấp khóa ngoại cho model ạ